src/three-components/ModelScene.js

```diff
@@ -195,14 +197,27 @@ export default class ModelScene extends Scene {
     this.canvas.style.height = `${this.height}px`;
     this.aspect = this.width / this.height;
 
-    const modelSize = this.model.size;
-    if (modelSize.x != 0 || modelSize.y != 0 || modelSize.z != 0) {
+    if (this.model.size.x != 0 || this.model.size.y != 0 ||
+        this.model.size.z != 0) {
+      const boxHalfX = Math.max(
+          Math.abs(this.model.boundingBox.min.x + this.model.position.x),
+          Math.abs(this.model.boundingBox.max.x + this.model.position.x));
```

Comment:
An age-old JS trick to cut down on minified code size is to destructure complex objects with deep paths. For example:

```javascript
const {boundingBox, position} = this.model;
const boxHalfX = Math.max(
          Math.abs(boundingBox.min.x + position.x),
          Math.abs(boundingBox.max.x + position.x));
```

In some cases, closure compiler with advanced optimizations can actually minify deep paths. But, I do not think we can currently compile this library with advanced optimizations.

Reply:
That's interesting. In this case, which would you prefer, smaller code size or fewer variables?

Reply:
I think for now I would prefer more identifiers because they are cheap and easily minified by non-fancy tools.